navidrome/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+services:
+  app_proxy:
+    environment:
+      APP_HOST: navidrome_server_1
+      APP_PORT: 4533
+      PROXY_AUTH_ADD: "false"
+  server:
+    image: deluan/navidrome:0.52.5@sha256:b154aebe8b33bae82c500ad0a3eb743e31da54c3bfb4e7cc3054b9a919b685c7
+    restart: on-failure
+    environment:
+      - PUID=1000
+      - PGID=1000
+    volumes:
+      - ${APP_DATA_DIR}/data:/data
```

Comment:
Please create a file called `navidrome/data/.gitkeep`.
The problem is that on older versions of umbrelOS the permissions of automatically created repositories are wrong. By supplying a .gitkeep file, those directories are getting created by installing the app automatically.

Reply:
now this worked, file created